redfish-core/lib/rde.hpp

```diff
@@ -238,14 +238,49 @@ inline void rdeGetHandler(const crow::Request& req,
 }
 
 // PATCH handler
-inline void rdePatchHandler(const crow::Request& /*req*/,
```

Comment:
Please provide some more background on this chabges . Also please include curl command used for the patch. in tested field.